src/updater.ts

```diff
@@ -1,15 +1,16 @@
-import bots from './bots';
+import getBots from './bots';
 import DataManager from './data_manager';
 import ConfigManager from './config_manager';
-import { games } from './game';
+import Game from './game';
 import botLogger from './bot_logger';
 import BotNotification from './notification';
 import RSS from './rss';
 import { sort, sortLimitEnd } from './comparable';
 
 const loggerTag = 'Updater';
 
-class Updater {
+export default class Updater {
+  private static updater: Updater;
```

Comment:
This does not work like Java, `static` is only for methods in JS. To create a singleton you either (assign the variable to the constructor](https://www.quora.com/How-do-you-create-static-variables-in-JavaScript) or you export an instance of the class instead of the class itself. Any script that imports will import the same instance effectively making it a singleton, and you can also freeze the export to make it more visible. Personally I prefer the second approach.

Reply:
This is a problem across the codebase and doesn't have much to do with the init check, so I moved this to #99.